updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -98,20 +98,22 @@ include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
 See xref:../../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object[Enabling the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
-[id="update-service-delete-service"]
-== Deleting an OpenShift Update Service application
+[id="nexts-steps_updating-restricted-network-cluster-osus"]
```

Comment:
Is this worth fixing? `nexts`

Reply:
Thanks for catching this, I'll fix it now